language-plutus-core/src/Language/PlutusCore/Lexer/Type.hs

```diff
@@ -123,6 +123,9 @@ data Token ann
     | EOF { loc :: ann }
     deriving (Show, Eq, Generic, NFData)
 
+instance Eq (Version ann) where
```

Comment:
Reference the note.

Reply:
Yeah, not that easy, because it's in a different file and the dependency goes the other way around. It never was a good idea to put core data types definitions under the `Lexer` folder (???).

Reply:
Just the note! i.e. `See whatever that note about not using annotations is`

Reply:
Yeah, did that.